pvacseq/lib/pipeline.py

```diff
@@ -57,6 +57,10 @@ def tsv_file_path(self):
     def convert_vcf(self):
         print("Converting VCF to TSV")
         sys.stdout.flush()
+        if os.path.exists(self.tsv_file_path()):
+            print("TSV file already exists. Skipping.")
+            return
```

Comment:
My preference would probably be to have these status messages on stderr.  If we were going to be more systematic about worrying about flushing or whatnot, then maybe it's time for a `def status_message` or even a logging framework?  But all of that goes way beyond this PR 😄